client/src/components/Posts/Hero.tsx

```diff
@@ -43,12 +42,19 @@ const Hero = () => {
       {
         //@ts-ignore
         posts.map((item, index) => (
-          <div key={index} className="flex justify-center items-center">
-            <Card
+          <div key={index} className="flex gap-4 justify-center items-center m-3">
+            {/* <Card
               link={item.ObjectURL}
               downloadIcon={<Downlaod />}
               filename={item.Filename}
+            /> */}
+            <ReactAudioPlayer            //added react audio player to preview audio files
```

Comment:
I'd recommend to use the `react-media-player` package since we have both audio and video files.